check_script.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+check_directory() {
+    if ! head -n 20 $FILE | grep -q "This program is free software: you can redistribute it and/or modify"
+    then
+        if ! head -n 20 $FILE | grep -q "Licensed under the Apache License, Version 2.0"
+        then
+            echo "$FILE does not contain a current copyright header"
+        fi
+    fi
+}
+
+
+for DIRS in owncloudApp/src owncloudData/src owncloudDomain/src owncloudTestUtil/src
+do
+    for FILE in $(find $DIRS -name "*.java" -o -name "*.kt")
+    do
+        check_directory
+    done
+done
+
+./gradlew ktlintFormat
+
+echo "========= Check Library =========="
+cd owncloud-android-library
+./check_script.sh
```

Comment:
Really interesting script to not forget to add the licenses header. 💯 

I think we should move it out of the root folder though. In both repositories, android and android-library. What do u think?

Reply:
And also, give it another name. `check_script` is not very descriptive. Something like `check_license_headers` or similar. About the location, not a problem for me to keep it in root, but also no strong opinion.